src/windows/config/index.html

```diff
@@ -1,79 +1,108 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>Mob Timer Configuration</title>
-    <link rel="stylesheet" href="./index.css">
-  </head>
-  <body>
-    <div class="container">
-      <div class="time">
-        <label for="minutes">
-          Each turn lasts:
-          <input type="number" id="minutes" min="1" class="minutes" /> minutes
+
+<head>
+  <meta charset="utf-8">
+  <title>Mob Timer Configuration</title>
+  <link rel="stylesheet" href="../../../node_modules/@ibm/plex/css/ibm-plex.min.css">
```

Comment:
Do we need to include the whole CSS from this package in order to get the fonts, or do we use more stuff from here?

Reply:
I guess this doesn't affect the app that negatively since it's already bundled though.

Reply:
Great input, I will switch to a font that isn't that big. The resulting mob timer zip got way bigger with IBM plex.